### PR TITLE
DOCSP-29066 Add Help Menu Context to UI Page

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

Add help menu context to UI page.

## STAGING

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-29066/projects/overview/

## JIRA

https://jira.mongodb.org/browse/DOCSP-29066

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65958590dcc0144a51b5fd49

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/mapping-rules/schema-mapping.txt [hunk 1]
```diff
@@ -82,7 +82,7 @@ objects for the migrated tables. The results look like this:
          "Quantity": 1,
          "Product": {
            "ProductID": 1,
-           "Name": "Mark Porter Action Figure",
+           "Name": "MongoDB 5.0 Action Figure",
```
Comment (ianf-mongodb) [inline source/mapping-rules/schema-mapping.txt:85]: **Ignore**

File: source/includes/fact-get-help-menu.rst [hunk 2]
```diff
@@ -0,0 +1,4 @@
+View links for documentation, contact support, community forums,
```
Comment (ajhuh-mdb) [inline source/includes/fact-get-help-menu.rst:1]: [suggestion] We want to make these list items parallel (same part of speech). Also looks like we missed "and" here.
```suggestion
View links for documentation, support, community forums, and
```

**Comment by ianf-mongodb** [comment]
> LGTM % a couple non-blocking nits/suggestions.

Done thanks for the suggestions @ajhuh-mdb

**Comment by ianf-mongodb** [comment]
Hi @tomhollander this poped up in a recent backlog refinement session, its still valid so I went ahead and added it to the UI overview page.

**Review by tomhollander — APPROVED** [review]

**Pull request merged**